src/app/frank/fd_frank_dedup.c

```diff
@@ -45,43 +27,41 @@ run( fd_frank_args_t * args ) {
   if( FD_UNLIKELY( !in_fseq ) ) FD_LOG_ERR(( "fd_alloca failed" ));
 
   ulong in_idx = 0UL;
-  for( fd_pod_iter_t iter = fd_pod_iter_init( verify_pods ); !fd_pod_iter_done( iter ); iter = fd_pod_iter_next( iter ) ) {
-    fd_pod_info_t info = fd_pod_iter_info( iter );
-    if( FD_UNLIKELY( info.val_type!=FD_POD_VAL_TYPE_SUBPOD ) ) continue;
-    char const  * verify_name =                info.key;
-    uchar const * verify_pod  = (uchar const *)info.val;
-
-    FD_LOG_INFO(( "joining firedancer.verify.%s.mcache", verify_name ));
-    in_mcache[ in_idx ] = fd_mcache_join( fd_wksp_pod_map( verify_pod, "mcache" ) );
+  for( ulong i=0; i<in_cnt; i++ ) {
+    char path[ 32 ];
+    snprintf( path, 32, "mcache%lu", i );
```

Comment:
stylistically what do you think of `snprintf( path, sizeof(path), "mcache%lu", i );`?